Description: Upstream changes introduced in version 2.6.0-ubuntu4
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 ossec-hids-server (2.6.0-ubuntu4) lucid; urgency=low
 .
   * Bug fix: bad copy of var/ossec/etc/ossec.conf
 .
 The person named in the Author field signed this changelog entry.
Author: Nicolas Zin <nicolas.zin@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- ossec-hids-server-2.6.0.orig/Makefile
+++ ossec-hids-server-2.6.0/Makefile
@@ -43,7 +43,7 @@ install:
 	cp -pr etc/client.keys ${DIR}/etc/ > /dev/null 2>&1 ||/bin/true
 	cp -pr src/agentlessd/scripts/* ${DIR}/agentless/
 	cp -pr etc/internal_options.conf ${DIR}/etc/
-	cp -pr etc/ossec-server.conf ${DIR}/etc/
+	cp -pr etc/ossec-server.conf ${DIR}/etc/ossec.conf
 	cp -pr src/rootcheck/db/*.txt ${DIR}/etc/shared/
 	cp -p active-response/*.sh ${DIR}/active-response/bin/
 	cp -p active-response/firewalls/*.sh ${DIR}/active-response/bin/
